docs/commands_in_detail/verify.md

```diff
@@ -0,0 +1,30 @@
+# `dtm verify` in Action
+
+The command `dtm verify` checks the following:
+
+## 1. Config File
+
+`dtm verify` first verifies if the config file can be loaded successfully.
+
+If not, the following information might be printed out:
+
+- if the config file doesn't exist, it reminds you if you forgot to specify the config file by using the "-f" parameter;
```

Comment:
If ?

Reply:
## Some common punctuation usages for "bullet points":

### Main Sentence with Sentence Fragments

Main sentence blabla:
- sentence fragment 1 without punctuation
- sentence fragment 1 without punctuation
- sentence fragment 3 without punctuation

### Main Sentence with Clauses

Main sentence blabla:
- clause 1 not ending with a period;
- clause 2 not ending with a period;
- clause 3 not ending with a period.

### Main Sentence with Full Sentences

Main sentence blabla:
- Full-sentence 1 ending with a period.
- Full-sentence 2 ending with a period.
- Full-sentence 3 ending with a period.

See [this post](https://www.linkedin.com/pulse/how-capitalize-punctuate-bullet-points-l-michelle-baker-phd/) for more information: 

I recommend [Dreyer's English](https://www.amazon.com/Dreyers-English-Utterly-Correct-Clarity/dp/0812995708) if you are interested in English writing :) I find it helpful and humourous.

Reply:
👍👍👍